src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Storage</PackageId>
 		<Description>Microsoft Azure Management Storage Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Storage</AssemblyName>
-		<Version>7.2.0-preview</Version>
+		<Version>8.0.0-preview</Version>
 		<PackageTags>Microsoft Azure Storage management;Storage;Storage management;</PackageTags>
 		<PackageReleaseNotes>See https://aka.ms/asdotnetsdkchangelog for release notes.</PackageReleaseNotes>
```

Comment:
Please confirm that the package release notes are updated

Reply:
nit: Is there a reason you plan to release a major version, you can release preview breaking changes by bumping the minor version too

Reply:
1. Where's the package release notes? Do you mean the change log?  I have updated it.
2. I update the major version just for the breaking change. Although we still in preview, we use the GA quality bar.